File: ATS.FrontEnd/ATS/src/app/candidato/models/candidato.ts
import { Curriculo } from "./curriculo";
import { Endereco } from "./endereco";

export class Candidato {
    id: string;
    nome: string;
    sobreNome: string;
    dataNascimento: Date;
    telefone: string;
    cpf: string;
    endereco: Endereco;
    curriculo: Curriculo;
}

File: ATS.BackEnd/ATS.BackEnd.Domain/Models/Candidato.cs
using System;

namespace ATS.BackEnd.Domain.Models
{
    public class Candidato : Entity
    {
        public string Nome { get; set; }
        public string SobreNome { get; set; }
        public string DataNascimento { get; set; }
        public string Telefone { get; set; }
        public string Cpf { get; set; }


        /* EF Relation */
        public Curriculo Curriculo { get; set; }
        public Endereco Endereco { get; set; }
    }
}
